src/controllers/users.js

```diff
@@ -12,6 +15,70 @@ class UsersController {
 
     return res.json(newUser);
   }
+
+  async deleteUser (req, res) {
+    try {
+      await User.deleteOne({ _id: req.params.id }).exec();
+
+      return res.status(200).json({ message: 'User is deleted.' });
+    } catch (error) {
```

Comment:
Return always status 200 (in case if there is no error during connection to db) and true/false in case user has been deleted/not deleted